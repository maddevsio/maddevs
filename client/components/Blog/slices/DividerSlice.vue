<template>
  <hr />
</template>

<script>
export default {
  name: 'Divider',
}
</script>

<style scoped lang="scss">
@import '@/assets/styles/_vars.scss';

hr {
  border-bottom: none;
  border-left: none;
  border-right: none;
  border-top: 1px solid $border-color--grey-20-percent;
  margin: 48px 0;
}
</style>
