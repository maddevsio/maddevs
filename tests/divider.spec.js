import { mount } from '@vue/test-utils'
import DividerSlice from '../client/components/Blog/slices/DividerSlice'

describe('divider slice component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(DividerSlice)
  })

  it('is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
