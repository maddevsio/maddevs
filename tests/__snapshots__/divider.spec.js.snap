// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divider slice component renders correctly 1`] = `<hr />`;
